python_modules/dagster/dagster/_core/definitions/declarative_scheduling/operands/parent_newer_condition.py

```diff
@@ -6,15 +6,25 @@
     SchedulingResult,
 )
 from dagster._core.definitions.declarative_scheduling.scheduling_context import SchedulingContext
+from dagster._core.definitions.declarative_scheduling.utils import DepKeySelection
 from dagster._core.definitions.events import AssetKeyPartitionKey
 from dagster._serdes.serdes import whitelist_for_serdes
 
 
 @whitelist_for_serdes
 class ParentNewerCondition(SchedulingCondition):
+    dep_selection: DepKeySelection = DepKeySelection.all()
+
     @property
     def description(self) -> str:
-        return "At least one parent has been updated more recently than the candidate."
+        description = "At least one parent has been updated more recently than the candidate"
+        if not self.dep_selection.is_all:
+            description += f" within selection: {self.dep_selection.description}"
+        return description
```

Comment:
nit: could be one-liner

Reply:
Maybe missing something dumb, but I'm not seeing a clean way to make this a oneliner, e.g.:

```python
return "All deps" + (f"within _selection: {self.dep_selection}" if self.dep_selection else "")
```

would work, but it feels a fair bit harder to read to me. Open to suggestions though